build.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -ex
+#DRY_RUN="--dryrun"
+
+if [ "$CODEBUILD_WEBHOOK_TRIGGER" != "branch/main" ]; then
```

Comment:
```suggestion
if [ "$CODEBUILD_WEBHOOK_TRIGGER" != "branch/master" ]; then
```

unless you want to change the default branch name of this repo

Reply:
Thanks for catching that.